README.rst

```diff
@@ -52,7 +52,18 @@ Under development.
 Usage
 -----
 
-Under development.
+In the root of auTO, create a config file named 'autoto.cfg'.
+It's contents **must** be the following:
```

Comment:
back to english class
the apostrophe is an exception here, not possessive but a contraction